plans/all.fmf

```diff
@@ -0,0 +1,46 @@
+adjust:
+    when: initiator is not defined or initiator != packit
+    discover+:
+        dist-git-source: true
+
+discover+:
+    how: fmf
+execute:
+    how: tmt
+
+/basic_check:
+    summary: Run basic checks
+    discover+:
+        test: /test/basic_check
```

Comment:
About these tests, can they be run sequentially one after another? If so it would make more sense to use a single plan with multiple tests, will also display nicer in `testing-farm`

Reply:
RE: sequentially, the system and e2e tests can run after the basic_check, but e2e and system tests I'd probably not want them run sequentially even if that's doable. For now, testing-farm does show 3 sections `basic, e2e and system`  though, which I guess is good enough

Reply:
Each test has it's own work directory, but they share the same machine, and the same environment produced by `plan.prepare`. That is mostly what I was referring on there. Do you want them to be completely independent, in which case, the original approach makes more sense, or do you want to reuse the `install` sections a bit to save up on some plan maintenance (having only upstream/downstream plans instead)